pkg/util/webhookpatch.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"encoding/json"
+	"fmt"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+	admissionregistrationv1beta1client "k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1"
+)
+
+// PatchMutatingWebhookConfig patches a CA bundle into the specified webhook config.
+func PatchMutatingWebhookConfig(client admissionregistrationv1beta1client.MutatingWebhookConfigurationInterface,
+	webhookConfigName, webhookName string, caBundle []byte) error {
+	config, err := client.Get(webhookConfigName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	old, err := json.Marshal(config)
+	if err != nil {
+		return err
+	}
+	found := false
+	for i, w := range config.Webhooks {
+		if w.Name == webhookName {
+			config.Webhooks[i].ClientConfig.CABundle = caBundle[:]
+			found = true
+			break
+		}
+	}
+	if !found {
+		return fmt.Errorf("webhook entry not found in config")
```

Comment:
Including the webhook name would be helpful here, e.g. `fmt.Errorf("webhook %q not found in %q", webhookName, webhookConfigName)`